tactical-microgrid-standard/common/opendds_tms_export.h

```diff
@@ -0,0 +1,57 @@
+
+// -*- C++ -*-
+// Definition for Win32 Export directives.
+// This file is generated automatically by generate_export_file.pl opendds_tms
+// ------------------------------
+#ifndef OPENDDS_TMS_EXPORT_H
+#define OPENDDS_TMS_EXPORT_H
+
+#include "ace/config-all.h"
+
+#if defined (ACE_AS_STATIC_LIBS) && !defined (OPENDDS_TMS_HAS_DLL)
+#  define OPENDDS_TMS_HAS_DLL 0
+#endif /* ACE_AS_STATIC_LIBS && OPENDDS_TMS_HAS_DLL */
+
+#if !defined (OPENDDS_TMS_HAS_DLL)
+#  define OPENDDS_TMS_HAS_DLL 1
+#endif /* ! OPENDDS_TMS_HAS_DLL */
+
+#if defined (OPENDDS_TMS_HAS_DLL) && (OPENDDS_TMS_HAS_DLL == 1)
+#  if defined (OPENDDS_TMS_BUILD_DLL)
+#    define opendds_tms_Export ACE_Proper_Export_Flag
```

Comment:
```suggestion
#    define OpenDDS_TMS_Export ACE_Proper_Export_Flag
```
(re-generate with the script)
These are usually all-caps or at least start with upper-case.

Reply:
If the export header is being committed, CMake will need to be explicitly told about the export header here:

https://github.com/OpenDDS/DeveloperResources/blob/505e6e090818cfa1885496f3846c84bb7bd0a55f/tactical-microgrid-standard/CMakeLists.txt#L23

Following the pattern we use in OpenDDS to do the same thing:

https://github.com/OpenDDS/OpenDDS/blob/9597435af567bd6e470dc97895a318eceae9fc60/dds/CMakeLists.txt#L583